test/fuzz/yaml/yaml.go

```diff
@@ -0,0 +1,32 @@
+package yaml
+
+import (
+	"gopkg.in/yaml.v2"
+	sigyaml "sigs.k8s.io/yaml"
+)
+
+func FuzzSigYaml(b []byte) int {
```

Comment:
the functions probably need export comments to pass go-lint.

Reply:
Done.